tox.ini

```diff
@@ -22,7 +22,8 @@ passenv = OS_* TEST_*
 deps =
     pyyaml
     pytest
-    charms.unit_test
+    #charms.unit_test
+    git+https://github.com/juju-solutions/charms.unit_test.git@feature/py310_support
```

Comment:
Don't Merge with this change!!

Reply:
Marking as draft so i remember not to merge with this change.  Feel free to review

Reply:
the [unit_test pr](https://github.com/juju-solutions/charms.unit_test/pull/19) is released as [v1.1.1](https://pypi.org/project/charms.unit-test/1.1.1/#files).  You can revert the `git+https` now if you'd like.